src/wasm.h

```diff
@@ -588,6 +588,9 @@ class Expression {
     assert(int(_id) == int(T::SpecificId));
     return (const T*)this;
   }
+
+  // Print the expression to stderr. Meant for use from debuggers.
```

Comment:
Nit: We can use this for printf-style debugging too, so it might not be strictly necessary to say this is for debuggers.

Reply:
Sure, I'll change it say "Meant for use while debugging."